packages/experimental/Button/SPEC.md

```diff
@@ -0,0 +1,191 @@
+# Button
```

Comment:
This file will replace the existing file in the docs folder once we move to the new button. Don't want to confuse people when we're still shipping the old button.